dynamiqs/utils/utils.py

```diff
@@ -25,7 +25,7 @@
     'braket',
     'overlap',
     'fidelity',
-    'mpow',
+    'mpow'
```

Comment:
Don't forget to run `task all`, on your last commit it will reformat the file (and add back this comma)
```shell
→ task black

>>> black
reformatted /Users/pierreguilmin/dev/github/dynamiqs/dynamiqs/utils/utils.py

All done! ✨ 🍰 ✨
1 file reformatted, 118 files left unchanged.
```